packages/e2e-tests/puppeteer/__tests__/rps.test.ts

```diff
@@ -42,7 +56,23 @@ describe('Playing a game of RPS', () => {
     expect(await rpsTabB.waitForXPath('//button[contains(., "Create a game")]')).toBeDefined();
     expect(await rpsTabA.waitForXPath('//button[contains(., "Create a game")]')).toBeDefined();
 
-    await clickThroughRPSUI(rpsTabA, rpsTabB);
+    await startAndFundRPSGame(rpsTabA, rpsTabB);
```

Comment:
This is arguably redundant with L72 : during the whole test suite we actually start 3 games now.  You could consider having test1: play a game, and close it out; test2: start a new game, raise and respond to a challenge?